UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/appsettings.json

```
@@ -1,10 +1,33 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
-    }
-  },
-  "AllowedHosts": "*"
+    "EventHubLoggingConfiguration": {
+        "MinimumLoggingLevel": "Warning",
+        "UkhoMinimumLoggingLevel": "Information",
+        "Environment": "dev",
+        "EntityPath": "",
+        "System": "ENS API",
```

Don't think it's an API:
```suggestion
        "System": "ENS",
```

---

Yes made changes accordingly.